server/matchmaker/search.py

```diff
@@ -14,6 +14,9 @@ class Search:
     """
     Represents the state of a users search for a match.
     """
+    noobFixBaseRating = 500 # The base rating for a noob with 0 games
```

Comment:
Let's stick with `snake_case` and perhaps use a less offensive name? E.g. `newbie_base_mean` and `newbie_game_threshold`.